docs_src/examples/Ch-ExamplesUsbDeviceService.md

```diff
@@ -0,0 +1,305 @@
+# USB Camera Device Service RTSP Streaming Guide
+## Overview
+The EdgeX usb device service is designed for communicating with USB cameras attached to Linux OS platforms. This guide will help configure and build the usb device service and start streaming video from the USB camera.
+
+This service provides the following capabilities:
+- V4L2 API to get camera metadata
+- Camera status
+- Video stream reference
+- FFmpeg framework to capture video frames and stream them to an RTSP server
```

Comment:
```suggestion
- Capture video frames and stream them to an RTSP server
```

Reply:
Why mark resolved with out comment why you didn't change?

Reply:
Sorry about that, I missed it. should be correct now.